components/Posts/PostEditor.test.tsx

```
@@ -0,0 +1,281 @@
+  isDestroyed: false,
+  isEditable: true,
+  on: jest.fn(),
+  off: jest.fn(),
+  setOptions: jest.fn(),
+};
+
+
+jest.mock('@tiptap/react', () => ({
+  ...jest.requireActual('@tiptap/react'),
+  useEditor: jest.fn(() => mockEditor),
+  EditorContent: jest.fn(({ editor }) => <div data-testid="editor-content" dangerouslySetInnerHTML={{ __html: editor?.getHTML() }}></div>),
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)